modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -29,12 +27,15 @@ ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
 endif::[]
 ifeval::["{context}" == "installing-platform-agnostic"]
 :agnostic:
+ifeval::["{context}" == "installing-rhv-restricted-network"]
```

Comment:
I believe the build issues are related to the missing `endif::[]` for the `agnostic` attribute here. It is also missing for the closing agnostic attribute at the bottom of this file.